test/tests/AverageExtensionRatio/AverageExtensionRatio.i

```diff
@@ -189,7 +187,7 @@
   [./stress]
     type = ComputeBeamResultants
     block = 0
-    outputs = exodus
+    outputs = csv
     output_properties = 'forces moments'
   [../]
 []
```

Comment:
I would prefer fixing the test by getting rid of the outputs=csv and adding these blocks.


```
[AuxVariables]
  [forces_x]
    order = CONSTANT
    family = MONOMIAL
  []
[]

[AuxKernels]
  [forces_x]
    type = MaterialRealVectorValueAux
    property = forces
    variable = forces_x
    execute_on = timestep_end
  []
[]
```

Reply:
I don't want to approve this without asking @bwspenc    
Maybe the output_properties  is useful so that we have some tests that will fail if the material property names change.  That should probably be checked by exodiffing these.

Reply:
Can confirm this change still gets the tests to pass

Reply:
Gone ahead and updated